### DISCUSSION TITLE
`Card` with footer slot

### DISCUSSION THREAD

**Discussion by zeshhaan** [element-plus/element-plus#6865] [Ideas]
https://element-plus.org/en-US/component/card.html#slots

Had a use case where i needed footer slot, kinda unfortunate that i can't use Card component just because its missing and i had to resort to make a custom component instead. 
Also i think its common nowadays that Card having some actions down the footer. So i would suggest having a footer slot for the Card component :)

Thanks in advance 🙏

**Comment by sxzz** [comment]
"Create issue from discussion" please

**Comment by zeshhaan** [comment]
Sure! The one linked to this discussion was closed automatically (because that was my first time doing that, might have missed something 😅) so manually created one at https://github.com/element-plus/element-plus/issues/6873
